kubernetes/README.md

```diff
@@ -16,6 +16,73 @@ Install the `dd-check-kubernetes` package manually or with your favorite configu
 
 Edit the `kubernetes.yaml` file to point to your server and port, set the masters to monitor
 
+### Gathering kubernetes events
+
+As the 5.17.0 release, Datadog Agent now supports built in leader election option for the Kubernetes event collector. Agents coordinate by performing a leader election among members of the Datadog DaemonSet through kubernetes to ensure only one leader agent instance is gathering events at a given time.
+If the leader agent instance fails, a re-election occurs and another cluster agent will take over collection.
+
+**This functionality is disabled by default**. 
+
+To enable leader election you need to set the variable `leader_candidate` to true in your kubernetes.yaml file.
+
+This feature relies on [ConfigMaps](https://kubernetes.io/docs/api-reference/v1.7/#configmap-v1-core) , so you will need to grant Datadog Agent get, list, delete and create access to the ConfigMap resource.
+
+Use this Kubernetes RBAC entities for your Datadog agent to properly configure the previous permissions buy [applying this datadog service account to your pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
```

Comment:
this Kubernetes RBAC entities --> these
buy --> by

Reply:
👍